provisioning/atat_provisioning.yaml

```
@@ -519,31 +515,32 @@ components:
         portfolio_id:
           type: string
           description: The ID of the Portfolio for which status is being returned
+        environment_id:
+          type: string
+          description: The ID of the Environment for which status is being returned
         status:
           type: string
           enum:
-            - "NOT_STARTED"
-            - "IN_PROGRESS"
-            - "COMPLETE"
-            - "FAILED"
-    PortfolioPatch:
-      properties:
-        administrators:
-          type: array
-          items:
-            $ref: '#/components/schemas/Operator'
-    Portfolio:
-      allOf:
-        - $ref: '#/components/schemas/PortfolioPatch'
+            - NOT_STARTED
+            - IN_PROGRESS
+            - COMPLETE
+            - FAILED
+    PortfolioCreate:
       description: >-
-        Logical container for a set of cloud resources which are paid for by a Task Order & its CLINs.
+        Version of Portfolio used for initial Portfolio creation. Does not include Environments.
```

Should this be something like
```suggestion
        Initial Portfolio created to contain Environments, which is an empty shell that does not initially include Environments.
```